autonomous/smokeTest.py

```diff
@@ -85,20 +86,23 @@ def runShooterMotors(self):
         """Stops the second hopper motor and runs both shooter motors for 2 seconds"""
         self.toDo = "Check to see if the shooter motors are running"
         #self.hopperMotor.stopHopperMotor2
-        self.shooterMotors.runShooter(sSpeed1 = self.dumbSpeed, sSpeed2 = self.dumbSpeed)
+        self.intakeMotor.stopIntake()
+        self.shooterMotors.runShooter(sSpeed1 = self.dumbRPMs, sSpeed2 = self.dumbRPMs)
+        self.shooterMotors.execute()
         log.error("Running both shooter motors")
 
     @state
     def calibrateTurret(self):
         """Calibrates the turret's deadzones and checks to see if the turret motor is working"""
         self.toDo = "Check to see if the turret is moving and that the deadzones are calibrated"
         self.shooterMotors.stopShooter()
-        self.next_state = "colorSensorCheck"
+        self.next_state("colorSensorCheck")
 
     @state
     def colorSensorCheck(self):
         """Has the user put up a red ball to the color sensor. Will not move on until the ball is red."""
```

Comment:
Can we add the exact same test for a blue ball too? Basically make them pass red and then make them pass blue?